lib/sunrise/sunrise.php

```diff
@@ -67,6 +67,7 @@ function handle_not_found_error( $error_type ) {
 	$is_web_request = Context::is_web_request();
 	if ( $is_web_request ) {
 		$is_maintenance_mode = Context::is_maintenance_mode();
+		$is_overdue_locked   = Context::is_overdue_locked();
```

Comment:
```suggestion
```

@vaurdan we have a stray line, now that we've removed the other code.

Reply:
Good catch! :D